CHANGES.rst

```diff
@@ -17,6 +17,16 @@ Bug fixes:
 - *add item here*
 
 
+2.0.1 (16.01.2019)
+------------------
+
+Bug fixes:
+
+- Fix broken tests in TravisCI
+  https://github.com/plone/plone.memoize/issues/12
+  [andbag]
+
+  
 2.0 (15.01.2019)
```

Comment:
Even this one does not seem to be there.

Reply:
Unfortunately I have no idea how the workflow from github to pypi works. Who is responsible for this step? At the moment the master branch is on version 2.0 and I only counted the version up. I can also add PR #14 to version 2.0.

@icemac What should I do?